src/Authorization.AspNetCore/IClaimsPrincipalAccessor.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System.Security.Claims;
+using GraphQL.Validation;
+
+namespace GraphQL.Server.Authorization.AspNetCore
+{
+    public interface IClaimsPrincipalAccessor
```

Comment:
I will be grateful if you can make even small, but still informative comments for all the added public APIs.